packages/app-form-builder/src/admin/components/FormEditor/Context/useFormEditorFactory.tsx

```diff
@@ -66,22 +59,36 @@ export interface FormEditor {
     ) => Promise<{ data: FbFormModel | null; error: FbErrorResponse | null }>;
     setData: (setter: SetDataCallable, saveForm?: boolean) => Promise<void>;
     getFields: () => FbFormModelField[];
-    getLayoutFields: (targetStepId: string) => FbFormModelField[][];
+    getStepFields: (targetStepId: string) => FbFormModelField[][];
     getField: (query: Partial<Record<keyof FbFormModelField, string>>) => FbFormModelField | null;
     getFieldPlugin: (
         query: Partial<Record<keyof FbBuilderFieldPlugin["field"], string>>
     ) => FbBuilderFieldPlugin | null;
-    insertField: (
-        field: FbFormModelField,
-        position: FieldLayoutPositionType,
-        targetStepId: string
-    ) => void;
-    moveField: (params: MoveFieldParams) => void;
+    insertField: ({
```

Comment:
Let's not inline types like this. Let's rather have `insertField: (params: InsertFieldParams) => void;`.

Above, around L48 I guess, you can then have:


```ts
export interface InsertFieldParams {
    data: FbFormModelField;
    target: DropTarget;
    destination: DropDestination;
}
```